From 323c4458c8967da5a4f7ba59c127d1280eecb2a5 Mon Sep 17 00:00:00 2001
From: Kosta Zertsekel <konszert@marvell.com>
Date: Mon, 22 Jul 2013 12:26:05 +0300
Subject: [PATCH 0863/1825] alp: pcie: use kzalloc instead of kcalloc

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit 87b17ec7ac3992f85229c4411fdd37297673a57e

	Rationale: kzalloc is more straight-forward and
	used virtually everywhere.

Signed-off-by: Kosta Zertsekel <konszert@marvell.com>
Change-Id: Ie84fe19fc96b033f14a749a5dfaa4c11776dc62f
Reviewed-on: http://vgitil04.il.marvell.com:8080/2753
Reviewed-by: Igor Patrik <igorp@marvell.com>
Tested-by: Igor Patrik <igorp@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-avantalp/pex.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-avantalp/pex.c b/arch/arm/mach-avantalp/pex.c
index 5f06589..4e5d4af 100644
--- a/arch/arm/mach-avantalp/pex.c
+++ b/arch/arm/mach-avantalp/pex.c
@@ -224,7 +224,7 @@ static int __init mv_pex_setup(int nr, struct pci_sys_data *sys)
 	if (mvCtrlPwrClckGet(PEX_UNIT_ID, nr) == MV_FALSE)
 		return 0;
 
-	res = kcalloc(1, sizeof(struct resource) * 2, GFP_KERNEL);
+	res = kzalloc(2 * sizeof(struct resource), GFP_KERNEL);
 	if (!res) {
 		panic("%s: memory alloc failed\n", __func__);
 		return 0;
-- 
1.7.5.4

